src/main/groovy/me/champeau/gradle/JapicmpAbstractTask.groovy

```diff
@@ -93,9 +98,11 @@ abstract class JapicmpAbstractTask extends AbstractTask {
         def options = new Options()
         options.outputOnlyModifications = onlyModified
         options.setAccessModifier(AccessModifier.valueOf(accessModifier.toUpperCase()))
+        options.htmlOutputFile = GOptional.fromNullable(htmlOutputFile?.absolutePath)
```

Comment:
I would like to avoid use of Guava as much as possible. The fact that `japicmp` brings it is already a bad thing IMHO and introduces conflicts with Gradle itself. Can you remove it?